src/test/java/edu/oakland/test/display01/DisplayGpsManagerImplementationTests.java

```diff
@@ -0,0 +1,56 @@
+package edu.oakland.test.display01;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import edu.oakland.helper.admin.Satellite;
+import edu.oakland.helper.display01.SatelliteSignalCheckRequest;
+import edu.oakland.production.display01.DisplayGpsManager;
+import edu.oakland.production.display01.DisplayGpsManagerImplementation;
+import edu.oakland.production.middleware01.MiddlewareGisInterface;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("Display GPS Manager Unit Tests")
+public class DisplayGpsManagerImplementationTests {
+  
+  @Test    
+  @DisplayName("Indication that GPS signal strengths are received.")
+  void receivedGpsSignalStrengths() {
+    Satellite sat = new Satellite("word", 3);
+    DisplayGpsManager man = new DisplayGpsManagerImplementation(
+        new MiddlewareGisInterfaceStub()
+    );
+    assertEquals("word", man.receiveGpsSignalStrength(sat));
+  }
+
+  @Test
+  @DisplayName("Indication that GPS signal strengths are received.")
+  void recheckGpsSignalStrength() {
+    Satellite sat = new Satellite("word", 3);
+    DisplayGpsManager man = new DisplayGpsManagerImplementation(
+        new MiddlewareGisInterfaceStub()
+    );
+    assertEquals("word", man.receiveGpsSignalStrength(sat));
```

Comment:
Here is the last part of what we were working on before the stream cut off:
```suggestion
    SatelliteSignalCheckRequest check = man.passGpsSignalStrength(sat);
    assertEquals("word", check.getSatelliteName());
    assertEquals("recheck", check.getCheckType());
```